tests/security/test_scan_for_secrets.py

```
@@ -0,0 +1,527 @@
+        docs_dir = tmp_path / "docs"
+        docs_dir.mkdir()
+        doc_file = docs_dir / "example.md"
+        doc_file.write_text("Example API key: sk-" + "A" * 48)
+
+        import os
+
+        original_dir = os.getcwd()
+        os.chdir(tmp_path)
+
+        try:
+            critical, warnings = scan_repository()
```

Ruff (0.14.4)

Unpacked variable `critical` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)